app/controllers/residential_address_controller.rb

```diff
@@ -21,6 +21,6 @@ def snap_application_param
   end
 
   def skip?
-    current_snap_application.mailing_address_same_as_residential_address?
+    current_snap_application&.mailing_address_same_as_residential_address?
```

Comment:
use the little lonely operator to check whether or not `current_snap_application` is nil or not before sending that next method.